app/views/diary_entries/_foster_logs.html.erb

```diff
@@ -0,0 +1,24 @@
+<%= render GovukComponent::SummaryList.new do |component| %>%
```

Comment:
I personally think this will be confusing as to where a one diary entry ends and another starts if these are separate elements of the `SummaryList` component. But we can deal with it later.